src/expr/src/foo/bar/mod.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
```

Comment:
Where is the corresponding .proto file?

I have to say that I am not fond of adding a random subcrate `foo` inside `mz-expr`. My top preference would be to have this exist in its own crate named `protobuf-example` or something like that.

At the minimum, can we have module documentation saying that this is supposed to give an example of rust type to protobuf type conversion?

Reply:
This is part of another commit that wasn't supposed to be part of this PR, feel free to remove it.

Reply:
Removed.